bin/tx-prover/README.md

```diff
@@ -81,6 +81,27 @@ This command will start the proxy using the workers defined in the configuration
 
 At the moment, when a worker added to the proxy stops working and can not connect to it for a request, the connection is marked as retriable meaning that the proxy will try reaching the following worker in a round-robin fashion. The amount of retries is configurable changing the `max_retries_per_request` value in the configuration file.
 
+## Updating workers on a running proxy
+
+To update the workers on a running proxy, you can use the `update-workers` command. This command will update the workers on the proxy and will not require a restart. To use this command, you will need to run:
+
+```bash
+miden-tx-prover update-workers [add|remove] [worker1] [worker2] ... [workerN]
```

Comment:
Why do we need to have an extra level of commands (i.e., `update-workers` and `add|remove`)? Would it make sense to simplify this as follows:
```
miden-tx-prover add-workers [worker1] [worker2] ... [workerN]
miden-tx-prover remove-workers [worker1] [worker2] ... [workerN]
```

Reply:
Yes, it makes sense. I used this approach, though internally I use the existing `UpdateWorker` structure and its execute method.